### ISSUE TITLE
Is jotta-cli download broken?

### ISSUE THREAD

**Issue by mabahj** [jotta/jotta-cli-issues#121]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release (`jotta-cli version`):**
```
jottad executable :
jottad appdata    : /var/lib/jottad
jottad version    : 0.7.33634
jotta-cli version : 0.7.33634
```
```
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.5 LTS
Release:        18.04
Codename:       bionic
```

**Description of problem:**
I get "ERROR  failed to start download" when downloading files. It worked just fine before upgrading to latest (0.7.33634) version. I had version  0.6.32724.
```
[zsh|0.26]% jotta-cli ls "Photos/Location Albums/Aix-en-Provence, France"
photo.jpg morephotos.jpg alotofphotos.jpg 
[zsh|0.22]% jotta-cli download "Photos/Location Albums/Aix-en-Provence, France" .
> starting download Photos/Location Albums/Aix-en-Provence, France => /home/me/temp/JottaPhotos_delete/Aix-en-Provence, France
> gathering metadata ... (ctrl-C to cancel)
> encountered error while gathering metadata..
ERROR  failed to start download: PathIsFile: File not found='/get-XXX/Jotta/Photos/Location Albums/Aix-en-Provence, France' in path='/photos/Location Albums/Aix-en-Provence, France'
XID: 370422934354
        Error 1: FILE_NOT_FOUND
[zsh|0.31|error 1]% 

```
**Expected:**
Downloading files working

**jotta-cli status (`jotta-cli status`):**
```
------------------------------------------------------------------------------
 Account   : myaccount on Jottacloud
 Usage     : 6.45TiB / ( Unlimited )
 Device    : myserver
 Backups   :
------------------------------------------------------------------------------
   Path      : /home/me/Dropbox
   Files     : 5463 files / 3.14GiB
   Status    : Up to date!

------------------------------------------------------------------------------
   Path      : /home/me/OneDrive
   Status    : Waiting for scan to start

------------------------------------------------------------------------------
   Path      : /media/backup
   Scanning  : /X/backup/X.hbk/Pool/0/1
   Files     : 62618 files / 702.93GiB

------------------------------------------------------------------------------
OK
```

**Relevant logs for the issue (`cat ~/.jottad/jottabackup.log`)**
```
pid:31369 2020/11/10 09:48:53 Error starting download: PathIsFile: File not found='/get-XX/Jotta/Photos/Location Albums/Aix-en-Provence, France' in path='/photos/Location Albums/Aix-en-Provence, France'
XID: 370422934354
        Error 1: FILE_NOT_FOUND
```

**Traceback**
```
N/A?
```

**Additional info:**
It is also behaving very strangely when downloading other files. I start downloading a file, it seems to accept that, but no file download is in progress (and no file is downloaded). I'm not able to start a new file download, because it says it is busy downloading. (Edit: I tried "jotta-cli list downloads" and that reports "Finished but 1 files (0bytes) could not be downloaded." and the log says "Giving up download after 4 retries.")
```
[zsh|1.01]% jotta-cli download Backup/shack/OneDrive/test-fra-shack.slettmeg .
> starting download Backup/shack/OneDrive/test-fra-shack.slettmeg => /home/me/temp/JottaPhotos_delete/test-fra-shack.slettmeg
> gathering metadata ... (ctrl-C to cancel)
> 1 files (0bytes) found in 240.611958ms
started download a22444facc
  run 'jotta-cli download --abort=a22444facc' to cancel
  run 'jotta-cli download --watch' to view progress
  or see 'jotta-cli list downloads' for information about
  all current, queued or failed downloads
OK
[zsh|1.01]% (I ran  jotta-cli download --watch)
No current downloads
^C

[zsh|1.01|error 130]% jotta-cli download Backup/shack/OneDrive/test-fra-shack.slettmeg .
> starting download Backup/shack/OneDrive/test-fra-shack.slettmeg => /home/me/temp/JottaPhotos_delete/test-fra-shack.slettmeg
> gathering metadata ... (ctrl-C to cancel)
> encountered error while gathering metadata..
ERROR  failed to start download: There already exists completed download with errors. Use jotta-cli download --abort=a22444facc to clear the previous download before retrying
[zsh|1.01|error 1]% jotta-cli download --abort=a22444facc 
OK
[zsh|1.01|error 1]%
```

**Comment by coopersimon** [comment]
Hi mabahj, thanks for your report. We've noticed similar issues with downloading photos ourselves and this should be fixed in the next release.

**Comment by Kimbsen** [comment]
We've just released version 0.7.35160 which should fix this issue.
